react-app/src/modules/account/account.test.js

```diff
@@ -0,0 +1,25 @@
+import * as account from './account.js';
+
+afterEach(() => account.clearAccount());
+
+test('creates keypairs from username and password', async () => {
+  expect.assertions(4);
+  await account.constructor('$/()RUFJ)wuwe84349', 'dnefu384(/·$873hf7g')
```

Comment:
Is there a reason to use such odd looking username? I think that for test purposes it could be enough just to use `"username"` and `"password"`.
Unless we specifically want to make sure that certain characters work, in which case I'd create a test specifying what we are testing.

Reply:
The reason was that this is one of the test vectors that was uploaded in the ticket. I can generate others if they are preferred.

Reply:
The idea is to test the functionality even for rare-characters, yes. If it is not necessarily a good practice to test it as the "normal" test, the problem is that these test take too much time, so I decided to check both things in one test. What do you think about this?

Reply:
yeah, it makes sense.